ibm_catalog.json

```diff
@@ -0,0 +1,193 @@
+                },
+                {
+                    "description": "The network ACL on this subnet grants all access from any source.\n",
+                    "title": "A subnet named 'client-to-site-subnet' in the landing-zone management VPC"
+                },
+                {
+                    "description": "A new security group named 'client-to-site-sg' that allows all incoming request from any source\n",
+                    "title": "client-to-site Security Group"
+                },
+                {
+                    "description": "An IAM access group allowing users to authenticate and connect to the client-to-site VPN gateway\n",
+                    "title": "A new IAM Access Group for VPN users"
+                },
+                {
+                    "description": "Using the private certificate generated above and stored in the Secret Manager instance, the gateway is located in the 'client-to-site-subnet' subnet mentioned, and attaches the 'client-to-site-sg' to the client-to-site VPN gateway with routes configured to allow accessing the landing zone VPCs (management and workload)\n",
+                    "title": "A client-to-site VPN server"
+                }
+            ],
+            "flavors": [
+                {
+                    "label": "Standard",
+                    "name": "standard",
+                    "install_type": "extension",
+                    "working_directory": "examples/landing-zone",
+                    "usage_template": "\n",
+                    "configuration": [
+                        {
+                            "hidden": false,
+                            "key": "vpc_id"
+                        },
+                        {
+                            "hidden": false,
+                            "key": "landing_zone_prefix"
+                        },
+                        {
+                            "key":"landing_zone_network_cidr",
+                            "hidden":false,
+                            "required":true
+                        },
+                        {
+                            "custom_config": {
+                                "config_constraints": {
+                                    "generationType": "2"
+                                },
+                                "grouping": "deployment",
+                                "original_grouping": "deployment",
+                                "type": "vpc_region"
+                            },
+                            "key": "region",
+                            "required": true
+                        },
+                        {
+                            "key": "ibmcloud_api_key",
+                            "required": true,
+                            "type": "password"
+                        },
+                        {
+                            "key": "resource_group"
+                        },
```

Comment:
There is no need to add any variables that we don't wish to customise or hide. So things like `resource_group`, `ibmcloud_api_key` can be removed from this JSON. Otherwise the list may become out of sync with what is in variables.tf